src/components/application_manager/src/policies/policy_handler.cc

```diff
@@ -587,16 +596,21 @@ void PolicyHandler::OnAppPermissionConsentInternal(
 
       out_permissions.policy_app_id = it->second;
       out_permissions.device_id = it->first;
-      policy_manager_->SetUserConsentForApp(out_permissions);
+#ifdef EXTERNAL_PROPRIETARY_MODE
+      if (!k_isNeedToUpdateExternalConsent) {
+#endif
```

Comment:
@pold500 I think it would be more readable: 
```
#ifdef EXTERNAL_PROPRIETARY_MODE
  if (!k_isNeedToUpdateExternalConsent) {
     policy_manager_->SetUserConsentForApp(out_permissions);
    }
#else 
    policy_manager_->SetUserConsentForApp(out_permissions);
#endif // EXTERNAL_PROPRIETARY_MODE
```